vendor/bat-native-ads/src/bat/ads/internal/ads/serving/eligible_ads/pipelines/inline_content_ads/eligible_inline_content_ads_factory.cc

```diff
@@ -28,6 +28,11 @@ std::unique_ptr<EligibleAdsBase> EligibleAdsFactory::Build(
                                              anti_targeting_resource);
     }
 
+    case 3: {
+      return std::make_unique<EligibleAdsV2>(subdivision_targeting,
```

Comment:
Just to doublecheck, this is suppose to be `V2` correct?

Reply:
This should be V3, even though we duplicated the code (for now until we refactor) we should still rename the class

Reply:
@tmancey do you want me to still make a v3 class and just replicate the v2 logic in there?

Reply:
Yes please

Reply:
@tmancey let me know if it's okay now